.github/actions/codemirror-e2e-tests/action.yaml

```diff
@@ -0,0 +1,34 @@
+name: 'Codemirror E2E tests'
+inputs:
+  browser:
+    description: 'Browser to run tests (chromium or firefox)'
+    required: true
+    default: 'chromium'
+
+runs:
+  using: 'composite'
+  steps:
+    - name: Validate browser input
+      shell: bash
+      run: |
+        if [[ "${{ inputs.browser }}" != "chromium" && "${{ inputs.browser }}" != "firefox" ]]; then
+          echo "Invalid browser (should be one of firefox or chromium): ${{ inputs.browser }}"
+          exit 1
+        fi
+    
+    - name: Install Playwright Browsers
+      shell: bash
+      run: pnpm exec playwright install --only-shell --with-deps
```

Comment:
it's good to specify firefox & chrome here, the default install includes webkit which makes the install take quite a bit longer